### DISCUSSION TITLE
Web cam putting

### DISCUSSION THREAD

**Discussion by RocknRollGolf** [springbok/MLM2PRO-GSPro-Connector#82] [General]
Can anyone tell me why i keep getting this message? I can't get the webcam putting to activate.  

![Screenshot 2024-03-04 162739](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/161998254/7d29ad89-3da3-4028-836f-b91820e0b547)

**Comment by springbok** [comment]
You could ask Alex as he is the developer of the putting app, you can find his github repo [here](https://github.com/alleexx/cam-putting-py)
